tests/unit_tests/test_dqc_us_0004.py

```diff
@@ -124,3 +123,37 @@ def test_values_unequal_very_unequal_values(self):
         self.assertTrue(_values_unequal(220, 100, -1))
         self.assertTrue(_values_unequal(600, 100, -2))
 
+    def test_values_scale_nan(self):
+        """
+        Make sure that the scale value is not valid if it is Not a Number
+        """
+        self.assert_true(_min_dec_valid(100,100,float('nan')))
```

Comment:
Not sure I understand why these _min_dec_valid calls are taking three parameters, when the function is only set up to test a single value.